doc/source/user_guide/skimage2_migration.md

```diff
@@ -0,0 +1,30 @@
+(skimage2_migration)=
+
+# Transition to skimage2
+
+:::{hint}
+This document is a work in progress and still subject to change.
+:::
+
+scikit-image is preparing to release version 2.0 under the new namespace `skimage2`.
+This will affect both its import name and its package name on PyPI and elsewhere.
```

Comment:
```suggestion
```

Reply:
I guess we will have an explicit section on how to install the new package and how to import it. Okay.